pyetrade/order.py

```diff
@@ -156,7 +156,10 @@ def build_order_payload(self, order_type, **kwargs):
         }
         order = kwargs
         order["Instrument"] = instrument
-        order["stopPrice"] = ""
+        if "stopPrice" in kwargs:
+          stopPrice = (float(kwargs["stopPrice"]) - 0.01)

```

Comment:
This is to properly convert float number to 2 decimal place dollar value 2 line below.